src/components/TopNav.tsx

```diff
@@ -1,20 +1,322 @@
+                                    <Link style={Links} to="/posts/learn">
+                                        Learn
+                                    </Link>
+                                </Button>
+                                <Button>
+                                    <Link style={Links} to="/posts/interview">
+                                        Interview
+                                    </Link>
+                                </Button>
+                                <Button>
+                                    <Link style={Links} to="/posts/project">
+                                        Project
+                                    </Link>
+                                </Button>
+                                <Button>
+                                    <Link style={Links} to="/login">
+                                        SignIn
+                                    </Link>
+                                </Button>
+                                <Button>
+                                    <Link style={Links} to="/register">
+                                        SignUp
+                                    </Link>
+                                </Button>
+                                <Button>
+                                    <Link style={Links} to="/my-account/1">
+                                        Profile
+                                    </Link>
+                                </Button>
+                                {toggleButton}
+                            </Box>
+                        </Responsive>
+                    </Toolbar>
+                </Container>
+            </AppBar>
+            <SmallView>
+            <Box component="nav">
+                <Drawer
+                    anchor={"right"}
+                    container={container}
+                    variant="temporary"
+                    open={mobileOpen}
+                    onClose={handleDrawerToggle}
+                    ModalProps={{
+                        keepMounted: true, // Better open performance on mobile.
+                    }}
+                    sx={style.drawerContent}
+                >
+                    {drawer}
+                </Drawer>
+            </Box>
+            </SmallView>
+        </>
     );
 };
 
-export default TopNav;
+const style = {
+    textCenter: {
+        textAlign: "center"
+    },
+    margin: {
+        iconBtn: { ml: .5 },
+        drawerTitle: { my: 2 }
+    },
+    flexSpace: {
+        between: {
+            justifyContent: "space-between",
+        }
+    },
+    drawerContent: {
+        "& .MuiDrawer-paper": {
+            boxSizing: "border-box",
+            width: 180,
+        },
+    },
+    typography: {        
+        mr: 2,
+        flexGrow: 1,
+        fontFamily: "monospace",
+        fontWeight: 700,
+        letterSpacing: ".2rem",
+        color: "inherit",
+        textDecoration: "none",        
+        small: {
```

Comment:
Alternative implementations for conditionally displaying this content has been outline in a previous comment